docs/releasenotes/release-1.124.md

```diff
@@ -1,24 +1,23 @@
 # v1.124.0
 
-** This version is not yet released; this document is gathering release notes for the future release **
+* Special shout-outs to @600lyy, @acpana, @anhdle-sso, @benjamin-maynard, @cheftako, @gemmahou, @hankfreund, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nancynh, @svetakvsundhar, @xiaoweim, @yuwenma for their contributions to this release.
 
-* ...
+## Announcement
 
-* Special shout-outs to ... for their
-  contributions to this release.
-TODO: list contributors with `git log v1.123.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+* We launched a major improvement to the Config Connector resource development!  Our new approach significantly enhances reliability and provides a more native Kubernetes experience. Learn more in our [guide](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/docs/develop-resources)   
 
-## Resources promoted from alpha to beta:
+* You can use the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on `ComputeForwardingRule` resource to opt-in
```

Comment:
`provides sustainable reconciliation support` is probably too focused on us, rather than the user.

It sounds like we are introducing functionality only in the direct reconciler?  I don't know if that's a great idea (because we can't indicate that in the CRD schema), but regardless we should focus on the user here.  Something like:

> This functionality requires setting the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation: spec.labels on kind=ServiceAttachment and kind=PrivateServiceConnect

Reply:
I would suggest leading with the result for the customer and then the steps they would need to take.

To use the Direct Cloud Reconciler for `ComputeForwardingRule`, set the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on `ComputeForwardingRule`. . .

Reply:
This is really helpful! Thank you. I restructured all the release note format to make it more user centric. Let me know  how it looks now